### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by ardias**
Udpate README following NR opensource template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# New Relic Infrastructure Integration for VMware vSphere
+[![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-New Relic Infrastructure Integration for VMware vShpere captures critical summary metrics and inventory data by connecting to the vCenter or to an ESXi Host. For more information, see [the official New Relic docs](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration).
+# New Relic VMware vShpere integration [![Known Vulnerabilities](https://snyk.io/test/github/newrelic/nri-vsphere/badge.svg?targetFile=vendor/vendor.json)](https://snyk.io/test/github/newrelic/nri-vsphere?targetFile=vendor/vendor.json)
```
Comment (theletterf) [inline README.md:3]: @ardias Is this supposed to be a button? I'd put it in the line below the title, not at the same level as the title.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
-vSphere data is obtained via a client provided by the `govmomi` package, the official go wrapper for VMware API. The same package provides a simulator for the virtual center, leveraged to run integration tests. The output of the integration is shaped by the `newrelic/infra-integrations-sdk` package and provided in JSON format. The agent collects such data and sends it to the New Relic infrastructure.
+New Relic's VMware vShpere integration captures critical summary and performance metrics data by connecting to VMware vCenter or an ESXi Host. For more information, see [the official New Relic docs](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration).
+
+The integration captures data about Datacenters, Clusters, Virtual Machines, Hosts, Datastores, ResourcePools and Networks. Apart from metrics, the integration also can capture vSphere events and VM snapshot information when enabled by the appropriate flags.
```
Comment (theletterf) [inline README.md:7]:
```suggestion
The integration collects data about data centers, clusters, virtual machines, hosts, datastores, resource pools, and networks. In addition to metrics, the integration can also capture vSphere events and VM snapshot information when enabled by the appropriate flags.
```

File: README.md [hunk 3]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
 
 ## Installation
 
-To install the integration, follow the official [documentation](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration). We recommend using a package manager.
+To install the integration, follow the official [documentation](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration). We recommend using your operating system native package manager.
```
Comment (theletterf) [inline README.md:16]:
```suggestion
To install the integration, follow the official [documentation](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration). We recommend using your operating system's package manager.
```

File: README.md [hunk 4]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+
+## Getting started
+
+After you've [installed](#installation) the integration you should make sure that the required configuration is appropriate for your environment.
```
Comment (theletterf) [inline README.md:20]:
```suggestion
After you've [installed](#installation) the integration make sure that you've the required configuration for your environment.
```

File: README.md [hunk 5]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+
+To configure the integration go to `/etc/newrelic-infra/integrations.d/` (Linux) or `C:\Program Files\New Relic\newrelic-infra\integrations.d\` (Windows) and open the file `vpshere-config.yml`.
```
Comment (theletterf) [inline README.md:22]:
```suggestion
To configure the integration go to `/etc/newrelic-infra/integrations.d/` (Linux) or `C:\Program Files\New Relic\newrelic-infra\integrations.d\` (Windows) and open the`vpshere-config.yml` configuration file.
```

File: README.md [hunk 6]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+
+You should configure the `URL`, `user` and `password` fields required to connect to you vCenter or ESXi host.
```
Comment (theletterf) [inline README.md:24]:
```suggestion
Configure the `URL`, `user`, and `password` fields -- they are required to connect to your vCenter or ESXi host.
```

File: README.md [hunk 7]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+
+To select which performance metrics to capture, the integration uses another file that you can use to select which performance metrics you want captured, per each `performance level` you require.
```
Comment (theletterf) [inline README.md:26]:
```suggestion
To select which performance metrics to capture, you must define them in the `vsphere-performance.metrics` file per each `performance level` you require.
```

File: README.md [hunk 8]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+You can find this file at `/etc/newrelic-infra/integrations.d/vsphere-performance.metrics` (Linux) or `C:\Program Files\New Relic\newrelic-infra\integrations.d\vsphere-performance.metrics` (Windows).
```
Comment (theletterf) [inline README.md:27]:
```suggestion
You can find this file in `/etc/newrelic-infra/integrations.d/vsphere-performance.metrics` (Linux) or `C:\Program Files\New Relic\newrelic-infra\integrations.d\vsphere-performance.metrics` (Windows).
```

File: README.md [hunk 9]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+Use the flag `--perf_level` to select which level of **performance metrics** you want to capture.
+
+Please note that the more **performance metrics** you enable the more load you will add to your envrionment.
```
Comment (theletterf) [inline README.md:30]:
```suggestion
Please note that the more performance metrics you enable the more load you add to your environment.
```

File: README.md [hunk 10]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
 
-## Development
+## Building
 
 If you have the source code and Go toolchain installed, you can build and run the vSphere integration locally.
```
Comment (theletterf) [inline README.md:34]:
```suggestion
If you have downloaded the source code and installed the Go toolchain, you can build and run the vSphere integration locally.
```

File: README.md [hunk 11]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
 
+vSphere data is obtained via a client provided by the `govmomi` package, the official go wrapper for VMware API. The same package provides a simulator for the virtual center, leveraged to run integration tests.
```
Comment (theletterf) [inline README.md:36]:
```suggestion
vSphere data is obtained via a client provided by the `govmomi` package, the official Go wrapper for the VMware API. The same package provides a simulator for the virtual center, leveraged to run integration tests.
```

File: README.md [hunk 12]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+
+The output of the integration is shaped by the `newrelic/infra-integrations-sdk` package and provided in JSON format. The New Relic Infra agent collects such data and sends it to New Relic.
```
Comment (theletterf) [inline README.md:38]:
```suggestion
The output of the integration is determined by the `newrelic/infra-integrations-sdk` package and provided in JSON format. The [New Relic Infrastructure agent](https://github.com/newrelic/infrastructure-agent) collects such data and sends it to New Relic.
```

File: README.md [hunk 13]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+
 1. After cloning this repository, go to vSphere integration directory and build the integration:
```
Comment (theletterf) [inline README.md:40]:
```suggestion
1. After cloning this repository, go to the vSphere integration directory and build the integration:
```

File: README.md [hunk 14]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
+    ```
+
 To learn more about the usage of `./bin/darwin/nri-vsphere`, pass the `-help` argument.
+
 ```bash
-$ ./bin/darwin/nri-vsphere -help
+./bin/darwin/nri-vsphere -help
 ```
 
 External dependencies are managed via govendor.
```
Comment (theletterf) [inline README.md:60]:
```suggestion
External dependencies are managed via `govendor`.
```

File: README.md [hunk 15]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
-
-## Custom integrations
-
-To extend your monitoring solution with custom metrics, we offer the Integrations Golang SDK, which can be found on [GitHub](https://github.com/newrelic/infra-integrations-sdk).
+```bash
+make test
+```
 
-Refer to [our docs site](https://docs.newrelic.com/docs/infrastructure/integrations-sdk/get-started/intro-infrastructure-integrations-sdk) to get help on how to build your custom integrations.
+You will need `docker-compose` to run the integration tests.
```
Comment (theletterf) [inline README.md:70]:
```suggestion
You need `docker-compose` to run the integration tests.
```

File: README.md [hunk 16]
```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
...
-
 New Relic hosts and moderates an online forum where customers can interact with New Relic employees as well as other customers to get help and share best practices. Like all official New Relic open source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this project's topic/threads here:
 
-https://discuss.newrelic.com/c/support-products-agents/new-relic-infrastructure
+[Community support thread](https://discuss.newrelic.com/t/new-relic-vmware-vsphere-integration/)
+
+## Contributing
 
-### Issues / Enhancement Requests
+We encourage your contributions to improv   e New Relic's VMware vShpere integration. Keep in mind when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You only have to sign the CLA one time per project.
```
Comment (davidbrota) [inline README.md:80]: typo -> improv   e

File: README.md (hunk 1 above)
Comment (ardias) [reply README.md:3]: Not a button, but an image (badge).
Makes sense to move below.

File: README.md (hunk 4 above)
Comment (ardias) [reply README.md:20]: hum.. not sure the contraction here reads well.. you mean "...that you have the..." right? I would not contract to "you've" then, reads a bit weird, no?

**Pull request merged**